### ISSUE TITLE
Can't directly switch from (M)ove to (R)otate and vice versa

### ISSUE THREAD

**Issue by repairfox** [facebook/Rapid#908]
### Description

My normal flow of operation:

1. Select RAPID building
2. Keyboard shortcut (A)dd
3. Keyboard shortcut (M)ove
4. Keyboard shortcut (R)otate
5. Repeat 4-5 as needed
6. Shift + select address
7. Keyboard shortcut (C)ombine

I can't switch from rotate to move, or move to rotate directly. After using 'M', I need to click somewhere on the map or 'esc', only after that can I select 'R'. This is only a problem since update, and adds multiple clicks to my operation.

### Version

2.0.1

### What browser are you seeing the problem on? What version are you running?

Chrome v111.0

### The OS you're using

win

### The browser URL at the time you encountered the bug

https://mapwith.ai/rapid#background=VBMPImagery_Most_Recent&datasets=msBuildings,a2b4bbc1bdbb4c8787a76ce787a948b8&disable_features=boundaries&map=19.96/38.22841/-78.86198&poweruser=true

### The auto-detected useragent string for your browser (leave blank if you're manually filling this form out)

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36

**Comment by bhousel** [comment]
Thanks, I agree! - this is a duplicate of #818
